models/User.js

```diff
@@ -123,6 +123,20 @@ const User = sequelize.define('user', {
         allowNull: true,
         defaultValue: ''
     },
+    primary_email: {
+        type: Sequelize.ENUM('personal', 'gsuite'),
+        allowNull: false,
+        defaultValue: 'personal'
+    },
+    notification_email: {
+        type: Sequelize.VIRTUAL,
+        get() {
+            if (this.primary_email === 'gsuite') {
+                return this.gsuite_id;
```

Comment:
do we have a column or field for it?

Reply:
@linuxbandit will include this in his PR for the GSuite wrapper

Reply:
At least according to [the issue of Jira](https://myaegee.atlassian.net/browse/MA-134)